src/data/move.ts

```diff
@@ -9155,13 +9163,13 @@ export function initMoves() {
       .attr(RechargeAttr),
     new AttackMove(Moves.SPECTRAL_THIEF, Type.GHOST, MoveCategory.PHYSICAL, 90, 100, 10, -1, 0, 7)
       .ignoresSubstitute()
-      .partial(),
+      .partial(), // Does not steal stats
     new AttackMove(Moves.SUNSTEEL_STRIKE, Type.STEEL, MoveCategory.PHYSICAL, 100, 100, 5, -1, 0, 7)
       .ignoresAbilities()
-      .partial(),
+      .edgeCase(), // Does not ignore abilities when called virtually (metronome)
     new AttackMove(Moves.MOONGEIST_BEAM, Type.GHOST, MoveCategory.SPECIAL, 100, 100, 5, -1, 0, 7)
       .ignoresAbilities()
-      .partial(),
+      .edgeCase(), // Does not ignore abilities when called virtually (metronome)
```

Comment:
```suggestion
      .edgeCase(), // Should not ignore abilities when called virtually (metronome)
```
Same as Sunsteel Strike.